build.sh

```diff
@@ -73,7 +73,7 @@ fi
 mkdir -p $WORKSPACE_DIR/build
 cd $WORKSPACE_DIR/build
 
-cmake .. -DCMAKE_BUILD_TYPE=$BUILD_TYPE && make -j$THREADS
+cmake .. -B./ -H../ -DCMAKE_BUILD_TYPE=$BUILD_TYPE && make -j$THREADS
```

Comment:
Hm... It wouldnt seem to compile on my linux distro. Not sure the reason, theoretically should work as its referencing the parent directory from the build dir but wouldnt seem to work on my build :/ idk if its my cmake or distro's or some configuration issue somewhere. Ill try figuring it out